### PR TITLE
Improve docs around packages and versions

### PR DESCRIPTION
**Pull request by bigmontz**

### CODE REVIEW

File: packages/neo4j-driver-lite/README.md [hunk 1]
```diff
@@ -1,7 +1,10 @@
-# Neo4j Driver for JavaScript
+# Neo4j Driver for JavaScript (lite)
 
 This is the lite version of the official Neo4j driver for JavaScript.
 
+This version of driver has the same capabilities of the Neo4j Driver except for the support reactive session.
```
Comment (AndyHeap-NeoTech) [inline packages/neo4j-driver-lite/README.md]: Slight grammar correction:

This version of _the_ driver has the same capabilities _as_ the Neo4j Driver except for the support _of_ reactive _sessions_

**Review by AndyHeap-NeoTech — APPROVED** [review]

File: packages/neo4j-driver-lite/README.md (hunk 1 above)
Comment (bigmontz) [inline packages/neo4j-driver-lite/README.md]:
```suggestion
This version of the driver has the same capabilities as the Neo4j Driver except for the support of reactive sessions.
```

File: packages/neo4j-driver-lite/README.md [hunk 2]
```diff
@@ -1,7 +1,10 @@
-# Neo4j Driver for JavaScript
+# Neo4j Driver for JavaScript (lite)
```
Comment (stefano-ottolenghi) [inline packages/neo4j-driver-lite/README.md]: I am unsure about the placement of `(lite)`: as it is, it looks like it refers to Javascript. Maybe `Neo4j Lite Driver for JavaScript`, or `Neo4j Driver Lite for JavaScript`? Applies a bit everywhere in this PR. 
Feel free to resolve with no action.

Comment (bigmontz) [inline packages/neo4j-driver-lite/README.md]:
```suggestion
# Neo4j Lite Driver for JavaScript
```

File: packages/neo4j-driver-lite/esdoc.json [hunk 3]
```diff
@@ -36,7 +36,7 @@
           "enable": false
         },
         "brand": {
-          "title": "Neo4j Bolt Driver 4.4 for JavaScript",
+          "title": "Neo4j Bolt Driver 5.x for JavaScript (lite)",
```
Comment (bigmontz) [inline packages/neo4j-driver-lite/esdoc.json]:
```suggestion
          "title": "Neo4j Bolt Lite Driver 5.x for JavaScript",
```

File: packages/neo4j-driver-lite/package.json [hunk 4]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "neo4j-driver-lite",
   "version": "5.0.0-dev",
-  "description": "Connect to Neo4j 3.5.0 and up from JavaScript",
+  "description": "The official Neo4j driver for Javascript (lite)",
```
Comment (bigmontz) [inline packages/neo4j-driver-lite/package.json]:
```suggestion
  "description": "The official Neo4j lite driver for Javascript",
```

File: packages/neo4j-driver-lite/README.md (hunk 2 above)
Comment (bigmontz) [reply packages/neo4j-driver-lite/README.md]: I've update with the `Neo4j lite driver` name

**Review by stefano-ottolenghi — APPROVED** [review]

**Pull request merged**